app/uk/gov/hmrc/incometaxsubscriptionselfemployedfrontend/config/AddressLookupConfig.scala

```diff
@@ -16,15 +16,15 @@
 
 package uk.gov.hmrc.incometaxsubscriptionselfemployedfrontend.config
 
-import play.api.i18n.{Lang, MessagesApi}
+import play.api.i18n.{Lang, Messages, MessagesApi}
```

Comment:
This import is not used
```suggestion
import play.api.i18n.{Lang, MessagesApi}
```